weblate/utils/markdown.py

```diff
@@ -58,8 +42,12 @@ def render_markdown(text):
         username = part[1:].lower()
         if username in users:
             user = users[username]
-            parts[pos] = '**[{}]({} "{}")**'.format(
-                part, user.get_absolute_url(), user.get_visible_name()
-            )
+            parts[
+                pos
+            ] = f'**[{part}]({user.get_absolute_url()} "{user.get_visible_name()}")**'
     text = "".join(parts)
-    return mark_safe(MARKDOWN(text))  # noqa: S308
+
+    # Initialize the mistletoe renderer
+    mistletoe_renderer = WeblateHtmlRenderer()
+
+    return mistletoe.markdown(text, renderer=mistletoe_renderer)
```

Comment:
This fails with TypeError: 'WeblateHtmlRenderer' object is not callable, see https://github.com/WeblateOrg/weblate/actions/runs/6518899707/job/17704819605?pr=10186#step:13:11688